httpcore5/src/main/java/org/apache/hc/core5/reactor/MultiCoreIOReactor.java

```diff
@@ -135,7 +135,7 @@ public final void close() {
 
     @Override
     public String toString() {
-        return super.toString() + " [status=" + status + "]";
```

Comment:
You should probably spit out at least the class name otherwise ot won't be really helpful. As far as I remember, other classes to the same.